tests/UsersTest.php

```diff
@@ -173,4 +173,101 @@ public function it_should_throw_a_generic_exception_when_client_suddenly_becomes
         // When
         $jira->users('123456789');
     }
+
+    /** @test */
+    public function it_should_return_the_profile_of_the_current_authenticated_user()
+    {
+        // Given
+        $jira = new Client(['clientId' => 1, 'clientSecret' => 'secret', 'redirectUrl' => 'none'], 'myorg', $this->token);
+
+        $jira->setClient($service = Mockery::mock('\GuzzleHttp\Client'));
+
+        $service->shouldReceive('request')
+            ->once()
+            ->andReturn(new Response(200, ['Content-Type' => 'application/json'], json_encode($this->user)));
+
+        // When
+        $users = $jira->myself();
```

Comment:
Shouldn't it be singular here?
` $user = $jira->myself();`

Reply:
Yes